### ISSUE TITLE
Pocket selection

### ISSUE THREAD

**Issue by TianmingQu** [Discngine/fpocket#160]
Hi, Is it ok that I do not use the extractISOPdb.py to extract the pocket grids and manually define a set of grids with resolution as 2A by myself?

Like for instance I set a grid map as 2A and use some geometric way to select out the grids that within the pocket region I am interested and then I use this manually defined grid-pdb  as input for MDpocket?

**Comment by pschmidtke** [comment]
yep that's fine, the extract ISOpdb thing is just there as a helper
